src/components/new-conversation-dialog/create-room/index.js

```diff
@@ -120,26 +121,28 @@ const CreateRoom = ({
         </Headline>
       </div>
       <form ref={ref} className={classes.form}>
-        <Switch
-          label={formatMessage({
-            id: 'ncdGroupPrivateLabel',
-            defaultMessage: 'Make group private',
-            description: 'switch label',
-          })}
-          aria-label={formatMessage({
-            id: 'ncdGroupPrivateLabel',
-            defaultMessage: 'Make group private',
-            description: 'switch label',
-          })}
-          onChange={setIsPrivate}
-          className={classes.switch}
-          helpText={formatMessage({
-            id: 'ncdGroupPrivateHelpText',
-            defaultMessage:
-              'Only group members can view a private group and invite other people to join.',
-            description: 'switch help text',
-          })}
-        />
+        {org.defaults.groupDefaults.visibility !== 'private' && (
```

Comment:
groupDefaults or groupDefaults.visibility might not be set.
we could use the `?.` operator to gracefully handle this  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Reply:
ok seems like this value is always returned from the server but I will still change this to be sure

Reply:
you can use `isPrivate` from props to toggle switch

Reply:
Also according to issue, I think it's incorrect to hide switch completely

Reply:
> Also according to issue, I think it's incorrect to hide switch completely

thx! fixed!